code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/MessageWebView.java

```diff
@@ -1,116 +0,0 @@
-/*
-  Copyright 2022 Adobe. All rights reserved.
-  This file is licensed to you under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License. You may obtain a copy
-  of the License at http://www.apache.org/licenses/LICENSE-2.0
-  Unless required by applicable law or agreed to in writing, software distributed under
-  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
-  OF ANY KIND, either express or implied. See the License for the specific language
-  governing permissions and limitations under the License.
-*/
-
-package com.adobe.marketing.mobile.services.ui;
-
-import android.content.Context;
-import android.graphics.Bitmap;
-import android.graphics.BitmapShader;
-import android.graphics.Canvas;
-import android.graphics.Color;
-import android.graphics.Paint;
-import android.graphics.PorterDuff;
-import android.graphics.PorterDuffXfermode;
-import android.graphics.RectF;
-import android.graphics.Shader;
-import android.webkit.WebView;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * An extension of {@link WebView} that customizes the drawing of the {@link Canvas} to allow for
- * rounded corners.
- */
-class MessageWebView extends WebView {
```

Comment:
did we check to make sure none of the other extensions aren't using `MessageWebView`?  acs or assurance?

Reply:
ACS and Target both use the AEPMessage class for displaying things in a webview. The webview is created internally within the AEPMessage / MessageFragment class though so removing the MessageWebView and using the standard WebView shouldn't cause dependency issues. I will setup tests to verify this.